operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md

```diff
@@ -26,6 +26,13 @@ echo $XNAME
 
 Only follow the steps in the section for the node type that is being rebuilt.
 
+> **`NOTE:`** (`ncn#`) Restart the `goss-servers` service on the rebuilt node after it has been rebuilt.
+> This is necessary because of a timing issue that has been fixed in CSM 1.6.1.
```

Comment:
```suggestion
> **`NOTE:`** (`ncn#`) If the CSM version is lower than 1.6.1, then restart the `goss-servers` service on the rebuilt node after it has been rebuilt.
> This is necessary because of a timing issue in the NCN's `cloud-init` which is fixed in CSM 1.6.1.
```